src/main/java/ru/spbstu/dto/VariantList.java

```diff
@@ -0,0 +1,19 @@
+package ru.spbstu.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import ru.spbstu.model.Variant;
+
+import java.util.List;
+
+public class VariantList {
+    @JsonProperty("variants")
+    public List<Variant> variants;
+
+    public List<Variant> getVariants() {
+        return variants;
+    }
+
+    public void setVariants(List<Variant> variants) {
+        this.variants = variants;
+    }
```

Comment:
The same about getters and setters.

Reply:
Used @Value lombok annotation so getter is created automagically, while setter is not created.